### PR TITLE
Remove all address book references

### LINKED ISSUE #82: Remove all references to Address and Tag

### PR DESCRIPTION
**Pull request by kailashgautham**
Closes #82

### CODE REVIEW

File: src/main/java/seedu/tatoolkit/model/person/ClassGroup.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class ClassGroup {
                     + "It cannot contain symbols, whitespaces and it should not be blank";
 
     /*
-     * The first character of the address must not be a whitespace,
+     * The first character of the tatoolkit must not be a whitespace,
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/ClassGroup.java:17]: This is meant to be class group i think

File: src/main/java/seedu/tatoolkit/model/person/Email.java [hunk 2]
```diff
@@ -1,10 +1,10 @@
-package seedu.address.model.person;
+package seedu.tatoolkit.model.person;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.tatoolkit.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's email in the address book.
+ * Represents a Person's email in the tatoolkit book.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Email.java]: Just tatookit

File: src/main/java/seedu/tatoolkit/model/person/Email.java [hunk 3]
```diff
@@ -36,7 +36,7 @@ public class Email {
     /**
      * Constructs an {@code Email}.
      *
-     * @param email A valid email address.
+     * @param email A valid email tatoolkit.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Email.java:39]: email address

File: src/main/java/seedu/tatoolkit/model/person/Github.java [hunk 4]
```diff
@@ -1,10 +1,10 @@
-package seedu.address.model.person;
+package seedu.tatoolkit.model.person;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.tatoolkit.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's GitHub ID in the address book.
+ * Represents a Person's GitHub ID in the tatoolkit book.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Github.java]: Just tatoolkit

File: src/main/java/seedu/tatoolkit/model/person/Github.java [hunk 5]
```diff
@@ -15,7 +15,7 @@ public class Github {
     public static final Github EMPTY = new Github("", true);
 
     /*
-     * The first character of the address must not be a whitespace,
+     * The first character of the tatoolkit must not be a whitespace,
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Github.java:18]: Github

File: src/main/java/seedu/tatoolkit/model/person/Name.java [hunk 6]
```diff
@@ -1,10 +1,10 @@
-package seedu.address.model.person;
+package seedu.tatoolkit.model.person;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.tatoolkit.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's name in the address book.
+ * Represents a Person's name in the tatoolkit book.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Name.java]: Just tatoolkit

File: src/main/java/seedu/tatoolkit/model/person/Name.java [hunk 7]
```diff
@@ -13,7 +13,7 @@ public class Name {
             "Names should only contain alphanumeric characters and spaces, and it should not be blank";
 
     /*
-     * The first character of the address must not be a whitespace,
+     * The first character of the tatoolkit must not be a whitespace,
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Name.java:16]: Name?

File: src/main/java/seedu/tatoolkit/model/person/Note.java [hunk 8]
```diff
@@ -1,10 +1,10 @@
-package seedu.address.model.person;
+package seedu.tatoolkit.model.person;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.tatoolkit.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's note in the address book.
+ * Represents a Person's note in the tatoolkit book.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Note.java]: Just tatoolkit

File: src/main/java/seedu/tatoolkit/model/person/Notes.java [hunk 9]
```diff
@@ -1,11 +1,11 @@
-package seedu.address.model.person;
+package seedu.tatoolkit.model.person;
 
 import static java.util.Objects.requireNonNull;
 
 import java.util.ArrayList;
 
 /**
- * Represents a Person's notes in the address book.
+ * Represents a Person's notes in the tatoolkit book.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Notes.java]: Just tatoolkit

File: src/main/java/seedu/tatoolkit/model/person/Person.java [hunk 10]
```diff
@@ -1,18 +1,18 @@
...
-import seedu.address.model.attendance.Attendance;
-import seedu.address.model.attendance.Week;
+import seedu.tatoolkit.commons.util.ToStringBuilder;
+import seedu.tatoolkit.model.attendance.Attendance;
+import seedu.tatoolkit.model.attendance.Week;
 
 
 /**
- * Represents a Person in the address book.
+ * Represents a Person in the tatoolkit book.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Person.java]: Just tatoolkit

File: src/main/java/seedu/tatoolkit/model/person/Phone.java [hunk 11]
```diff
@@ -1,10 +1,10 @@
-package seedu.address.model.person;
+package seedu.tatoolkit.model.person;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.tatoolkit.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's phone number in the address book.
+ * Represents a Person's phone number in the tatoolkit book.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Phone.java]: Just tatoolkit

File: src/main/java/seedu/tatoolkit/model/person/Telegram.java [hunk 12]
```diff
@@ -1,10 +1,10 @@
-package seedu.address.model.person;
+package seedu.tatoolkit.model.person;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.tatoolkit.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's Telegram ID in the address book.
+ * Represents a Person's Telegram ID in the tatoolkit book.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Telegram.java]: Just tatoolkit

File: src/main/java/seedu/tatoolkit/model/person/Telegram.java [hunk 13]
```diff
@@ -16,7 +16,7 @@ public class Telegram {
     public static final Telegram EMPTY = new Telegram("", true);
 
     /*
-     * The first character of the address must not be a whitespace,
+     * The first character of the tatoolkit must not be a whitespace,
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/model/person/Telegram.java:19]: telegram

File: src/main/java/seedu/tatoolkit/storage/JsonSerializableTaToolkit.java [hunk 14]
```diff
@@ -8,53 +8,53 @@
...
+     * @param source future changes to this will not affect the created {@code JsonSerializableTaToolkit}.
      */
-    public JsonSerializableAddressBook(ReadOnlyAddressBook source) {
+    public JsonSerializableTaToolkit(ReadOnlyTaToolkit source) {
         persons.addAll(source.getPersonList().stream().map(JsonAdaptedPerson::new).collect(Collectors.toList()));
     }
 
     /**
-     * Converts this address book into the model's {@code AddressBook} object.
+     * Converts this tatoolkit book into the model's {@code TaToolkit} object.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/storage/JsonSerializableTaToolkit.java]: Just tatoolkit

File: src/test/java/seedu/tatoolkit/model/person/NameContainsKeywordsPredicateTest.java [hunk 15]
```diff
@@ -68,7 +68,7 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         predicate = new NameContainsKeywordsPredicate(Arrays.asList("Carol"));
         assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").build()));
 
-        // Keywords match phone, email and address, but does not match name
+        // Keywords match phone, email and tatoolkit, but does not match name
```
Comment (g-tejas) [inline src/test/java/seedu/tatoolkit/model/person/NameContainsKeywordsPredicateTest.java:71]: This should be telegram or something?

File: src/main/java/seedu/tatoolkit/logic/commands/ClearCommand.java [hunk 16]
```diff
@@ -1,23 +1,23 @@
...
 import static java.util.Objects.requireNonNull;
 
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
+import seedu.tatoolkit.model.Model;
+import seedu.tatoolkit.model.TaToolkit;
 
 /**
- * Clears the address book.
+ * Clears the tatoolkit book.
```
Comment (g-tejas) [inline src/main/java/seedu/tatoolkit/logic/commands/ClearCommand.java]: Just Ta tookit

**Review by g-tejas — CHANGES_REQUESTED** [review]
Found a few parts that need to be changed

File: copyright.txt [hunk 17]
```diff
@@ -1,8 +1,7 @@
 ﻿Some code adapted from http://code.makery.ch/library/javafx-8-tutorial/ by Marco Jakob
 
 Copyright by Susumu Yoshida - http://www.mcdodesign.com/
-- address_book_32.png
-- AddressApp.ico
+- ta_toolkit_32.png
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply copyright.txt:4]: Oh I noticed that the .ico file does not exist in our repo. Can we remove it from the copyright file then?

File: src/main/java/seedu/tatoolkit/logic/commands/ClearCommand.java (hunk 16 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/logic/commands/ClearCommand.java]: Done!

File: src/test/java/seedu/tatoolkit/model/person/NameContainsKeywordsPredicateTest.java (hunk 15 above)
Comment (kailashgautham) [reply src/test/java/seedu/tatoolkit/model/person/NameContainsKeywordsPredicateTest.java:71]: Fixed!

File: src/main/java/seedu/tatoolkit/storage/JsonSerializableTaToolkit.java (hunk 14 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/storage/JsonSerializableTaToolkit.java]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Telegram.java (hunk 13 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Telegram.java:19]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Telegram.java (hunk 12 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Telegram.java]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Phone.java (hunk 11 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Phone.java]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Person.java (hunk 10 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Person.java]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Notes.java (hunk 9 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Notes.java]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Note.java (hunk 8 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Note.java]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Name.java (hunk 6 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Name.java]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Github.java (hunk 5 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Github.java:18]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Github.java (hunk 4 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Github.java]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Email.java (hunk 3 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Email.java:39]: Done!

File: src/main/java/seedu/tatoolkit/model/person/Email.java (hunk 2 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/Email.java]: Done!

File: src/main/java/seedu/tatoolkit/model/person/ClassGroup.java (hunk 1 above)
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/person/ClassGroup.java:17]: Done!

File: src/main/java/seedu/tatoolkit/model/TaToolkit.java [hunk 18]
```diff
@@ -60,24 +60,24 @@ public void resetData(ReadOnlyAddressBook newData) {
...
     public boolean hasPerson(Person person) {
         requireNonNull(person);
         return persons.contains(person);
     }
 
     /**
-     * Adds a person to the address book.
-     * The person must not already exist in the address book.
+     * Adds a person to the tatoolkit book.
+     * The person must not already exist in the tatoolkit book.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/TaToolkit.java]: Done!

File: src/main/java/seedu/tatoolkit/model/ModelManager.java [hunk 19]
```diff
@@ -10,37 +10,37 @@
...
-import seedu.address.commons.core.GuiSettings;
-import seedu.address.commons.core.LogsCenter;
-import seedu.address.model.person.Person;
+import seedu.tatoolkit.commons.core.GuiSettings;
+import seedu.tatoolkit.commons.core.LogsCenter;
+import seedu.tatoolkit.model.person.Person;
 
 /**
- * Represents the in-memory model of the address book data.
+ * Represents the in-memory model of the tatoolkit book data.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/ModelManager.java]: Done!

File: src/main/java/seedu/tatoolkit/model/Model.java [hunk 20]
```diff
@@ -36,43 +36,43 @@ public interface Model {
     void setGuiSettings(GuiSettings guiSettings);
 
     /**
-     * Returns the user prefs' address book file path.
+     * Returns the user prefs' tatoolkit book file path.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/model/Model.java]: Done!

File: src/main/java/seedu/tatoolkit/logic/commands/ViewCommand.java [hunk 21]
```diff
@@ -1,21 +1,21 @@
...
+import seedu.tatoolkit.commons.core.index.Index;
+import seedu.tatoolkit.commons.util.ToStringBuilder;
+import seedu.tatoolkit.logic.Messages;
+import seedu.tatoolkit.logic.commands.exceptions.CommandException;
+import seedu.tatoolkit.model.Model;
+import seedu.tatoolkit.model.person.Person;
 
 /**
- * Views details of a person in the address book identified using its displayed index from the address book.
+ * Views details of a person in the tatoolkit book identified using its displayed index from the tatoolkit book.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/logic/commands/ViewCommand.java]: Done!

File: src/main/java/seedu/tatoolkit/logic/commands/MarkCommand.java [hunk 22]
```diff
@@ -1,25 +1,25 @@
...
+import seedu.tatoolkit.commons.util.ToStringBuilder;
+import seedu.tatoolkit.logic.Messages;
+import seedu.tatoolkit.logic.commands.exceptions.CommandException;
+import seedu.tatoolkit.model.Model;
+import seedu.tatoolkit.model.attendance.Week;
+import seedu.tatoolkit.model.person.Person;
 
 /**
- * Adds a person to the address book.
+ * Adds a person to the tatoolkit book.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/logic/commands/MarkCommand.java]: Done!

File: src/main/java/seedu/tatoolkit/logic/commands/FindCommand.java [hunk 23]
```diff
@@ -1,14 +1,14 @@
...
-import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.tatoolkit.commons.util.ToStringBuilder;
+import seedu.tatoolkit.logic.Messages;
+import seedu.tatoolkit.model.Model;
+import seedu.tatoolkit.model.person.NameContainsKeywordsPredicate;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Finds and lists all persons in tatoolkit book whose name contains any of the argument keywords.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/logic/commands/FindCommand.java]: Done!

File: src/main/java/seedu/tatoolkit/logic/commands/EditCommand.java [hunk 24]
```diff
@@ -1,38 +1,38 @@
...
+import seedu.tatoolkit.model.person.Github;
+import seedu.tatoolkit.model.person.Name;
+import seedu.tatoolkit.model.person.Notes;
+import seedu.tatoolkit.model.person.Person;
+import seedu.tatoolkit.model.person.Phone;
+import seedu.tatoolkit.model.person.Telegram;
 
 /**
- * Edits the details of an existing person in the address book.
+ * Edits the details of an existing person in the tatoolkit book.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/logic/commands/EditCommand.java]: Done!

File: src/main/java/seedu/tatoolkit/logic/commands/DeleteNoteCommand.java [hunk 25]
```diff
@@ -1,23 +1,23 @@
...
+import seedu.tatoolkit.logic.Messages;
+import seedu.tatoolkit.logic.commands.exceptions.CommandException;
+import seedu.tatoolkit.model.Model;
+import seedu.tatoolkit.model.person.Note;
+import seedu.tatoolkit.model.person.Person;
 
 /**
- * Deletes notes for a person identified using the person's displayed index from the address book.
+ * Deletes notes for a person identified using the person's displayed index from the tatoolkit book.
  * The note to delete is identified by its index number in the side panel.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/logic/commands/DeleteNoteCommand.java]: Done!

File: src/main/java/seedu/tatoolkit/logic/commands/DeleteCommand.java [hunk 26]
```diff
@@ -1,18 +1,18 @@
...
+import seedu.tatoolkit.commons.core.index.Index;
+import seedu.tatoolkit.commons.util.ToStringBuilder;
+import seedu.tatoolkit.logic.Messages;
+import seedu.tatoolkit.logic.commands.exceptions.CommandException;
+import seedu.tatoolkit.model.Model;
+import seedu.tatoolkit.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a person identified using it's displayed index from the tatoolkit book.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/logic/commands/DeleteCommand.java]: Done!

File: src/main/java/seedu/tatoolkit/logic/commands/AddNoteCommand.java [hunk 27]
```diff
@@ -1,22 +1,22 @@
...
+import seedu.tatoolkit.commons.util.ToStringBuilder;
+import seedu.tatoolkit.logic.Messages;
+import seedu.tatoolkit.logic.commands.exceptions.CommandException;
+import seedu.tatoolkit.model.Model;
+import seedu.tatoolkit.model.person.Note;
+import seedu.tatoolkit.model.person.Person;
 
 /**
- * Adds a note to a person identified using it's displayed index from the address book.
+ * Adds a note to a person identified using it's displayed index from the tatoolkit book.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/logic/commands/AddNoteCommand.java]: Done!

File: src/main/java/seedu/tatoolkit/logic/commands/AddCommand.java [hunk 28]
```diff
@@ -1,25 +1,25 @@
...
+import seedu.tatoolkit.commons.core.LogsCenter;
+import seedu.tatoolkit.commons.util.ToStringBuilder;
+import seedu.tatoolkit.logic.Messages;
+import seedu.tatoolkit.logic.commands.exceptions.CommandException;
+import seedu.tatoolkit.model.Model;
+import seedu.tatoolkit.model.person.Person;
 
 /**
- * Adds a person to the address book.
+ * Adds a person to the tatoolkit book.
```
[the remark this answers is not in the document]
Comment (kailashgautham) [reply src/main/java/seedu/tatoolkit/logic/commands/AddCommand.java]: Done!

**Review by g-tejas — APPROVED** [review]
LGTM!

**Review by YuSoonZ — APPROVED** [review]
LGTM!

**Review by teojunda — APPROVED** [review]

**Review by hiivan — APPROVED** [review]
LGTM

**Pull request merged**